win32/Makefile

```diff
@@ -0,0 +1,80 @@
+#

```

Comment:
Could you put your copyright notice here?

Reply:
I've added the copyright notice.
I also update the comment in this header.